src/trivia/config.h.cmake

```diff
@@ -290,6 +290,7 @@
 #cmakedefine ENABLE_ETCD_CLIENT 1
 #cmakedefine ENABLE_CONFIG_EXTRAS 1
 #cmakedefine ENABLE_FAILOVER 1
+#cmakedefine ENABLE_INTEGRITY 1
```

Comment:
Minor: You can use `#cmakedefine01 ENABLE_INTEGRITY` to use `#if` instead of `#ifdef`. Feel free to ignore.

Reply:
Ignored for consistency reasons.